### ISSUE TITLE
Non-working folder ignore button (only in untrusted)

### ISSUE THREAD

**Issue by tomasz1986** [syncthing/syncthing#7258]
The folder ignore button seems non-functional in v1.12.1.

0. Enable the untrusted GUI.
1. Create a new folder on Device A.
2. Share the newly created folder with Device B.
3. Try to ignore the folder on Device B.
4. The ignore button does nothing.

```
ReferenceError: id is not defined
    at Scope.$scope.ignoreFolder (syncthingController.js:1984)
    at $parseFunctionCall (angular.js:12493)
    at callback (angular.js:21719)
    at Scope.$eval (angular.js:14589)
    at Scope.$apply (angular.js:14688)
    at HTMLButtonElement.<anonymous> (angular.js:21724)
    at HTMLButtonElement.dispatch (jquery-2.2.2.js:4755)
    at HTMLButtonElement.elemData.handle (jquery-2.2.2.js:4567)
```

The workaround is to manually remove the entry from `config.xml`.

Tested with v1.12.1.

**Comment by acolomb** [comment]
This was fixed in the default GUI in #7224.  The original merge conflict was apparently mirrored to the untrusted GUI as well.

**Comment by acolomb** [comment]
Actually that wasn't quite accurate.  The fix in #7224 was made after v1.12.1 had been branched.  The original error was introduced in #7055 already, and I noticed it on the main branch afterwards.  So this is already fixed and will be in the next rc.

I don't think it's important enough to warrant a hotfix release, because the untrusted GUI is only a preview anyway?
